website/app/styles/doc-components/font-helpers-list.scss

```diff
@@ -1,36 +1,58 @@
 // DOC FONT HELPERS LIST
 
 @use "../typography/mixins" as *;
+@use "../breakpoints" as breakpoint;
 
 .doc-font-helpers-list {
-  display: grid;
-  grid-template-columns: minmax(120px, max-content) auto;
+  display: flex;
+  flex-direction: column;
   gap: 16px 0;
   margin: 1.5rem 0;
   padding: 0;
   list-style: none;
+
+  @include breakpoint.medium-and-above () {
+    display: grid;
+    grid-template-columns: minmax(120px, max-content) auto;
+  }
 }
 
 .doc-font-helpers-list__item {
-  display: contents;
+  display: flex;
+  flex-direction: column;
+
+  @include breakpoint.medium-and-above () {
+    display: contents;
+  }
 }
 
 .doc-font-helpers-list__preview {
   display: flex;
   align-items: center;
   align-self: stretch;
-  justify-content: center;
+  justify-content: flex-start;
   min-width: 120px;
   padding: 16px;
   border: 1px solid var(--doc-color-gray-500);
-  border-radius: 3px 0 0 3px;
+  border-radius: 3px 3px 0 0;
+
+  @include breakpoint.medium-and-above () {
+    justify-content: center;
+    text-align: center;
+    border-radius: 3px 0 0 3px;
+  }
 }
 
 .doc-font-helpers-list__content {
   padding: 16px;
-  border: 1px solid var(--doc-color-gray-500);
-  border-left: none;
-  border-radius: 0 3px 0 3px;
+  border: solid var(--doc-color-gray-500);
```

Comment:
Nitpick: this declaration in interpreted by the browser as
```
border: initial solid var(--doc-color-gray-500);
```
(where the `initial` value in the browser is usually `1px`)

What you intend to do here, I think, is:
```
border-style: solid;
border-color: var(--doc-color-gray-500);
```